spec/features/10_user_searches_for_food_truck_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'rails_helper'
+RSpec.describe "User visits trucks search page" do
+  before(:each) do
+    copley = Location.create(body: "Copley Square")
```

Comment:
here in your tests, it may be best to use `create!` - you'd want your test to error out if validation fails.